ftd/fpm.ftd

```diff
@@ -400,3 +400,16 @@ option integer self-version:
 
 
 -- tracking-info list tracks:
+
+
+;; FPM Apps Installation
+
+-- record app-data:
+optional caption name:
```

Comment:
Not optional.

Reply:
Let's rename this to `mount-point`. There is no `name` for an app, just the mount-point.

Reply:
https://github.com/ftd-lang/fpm/discussions/599#discussioncomment-4076327